arch/x86/configs/eve_defconfig

```diff
@@ -731,17 +732,18 @@ CONFIG_PINCTRL_CHERRYVIEW=y
 CONFIG_PINCTRL_BROXTON=y
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_PCA953X=m
+CONFIG_GPIO_KEMPLD=m
 CONFIG_SENSORS_NCT6775=m
 # CONFIG_THERMAL_HWMON is not set
 CONFIG_WATCHDOG=y
-CONFIG_WATCHDOG_CORE=y
```

Comment:
@rucoder are you sure we can remove this one?

Reply:
@rene yes, because this feature is autoatically enabled when KEMPLD_WD is enabled. this is how defconfig minimizes the number of entries